resources/i18n/it_IT/cura.po

```diff
@@ -5616,33 +4927,20 @@ msgid "Yes"
 msgstr "Sì"
 
 msgctxt "@label"
-msgid ""
-"You are about to remove all printers from Cura. This action cannot be undone.\n"
-"Are you sure you want to continue?"
-msgstr ""
-"Si stanno per rimuovere tutte le stampanti da Cura. Questa azione non può essere annullata. \n"
+msgid "You are about to remove all printers from Cura. This action cannot be undone.\nAre you sure you want to continue?"
+msgstr "Si stanno per rimuovere tutte le stampanti da Cura. Questa azione non può essere annullata. "
 "Continuare?"
 
-#, python-brace-format
 msgctxt "@label"
-msgid ""
-"You are about to remove {0} printer from Cura. This action cannot be undone.\n"
-"Are you sure you want to continue?"
-msgid_plural ""
-"You are about to remove {0} printers from Cura. This action cannot be undone.\n"
-"Are you sure you want to continue?"
-msgstr[0] ""
-"Si sta per rimuovere {0} stampante da Cura. Questa azione non può essere annullata.\n"
-"Continuare?"
-msgstr[1] ""
-"Si stanno per rimuovere {0} stampanti da Cura. Questa azione non può essere annullata.\n"
-"Continuare?"
+msgid "You are about to remove {0} printer from Cura. This action cannot be undone.\nAre you sure you want to continue?"
+msgid_plural "You are about to remove {0} printers from Cura. This action cannot be undone.\nAre you sure you want to continue?"
+msgstr[0] "Si sta per rimuovere {0} stampante da Cura. Questa azione non può essere annullata.\nContinuare?"
```

Comment:
"Si sta per rimuovere la stampante {0} da Cura"...

Reply:
Same for plural ?

Reply:
plural is "Si stanno per rimuovere le stampanti {0} da Cura"